### PR TITLE
feat: added cart and wishlist page

### PR DESCRIPTION
**Pull request by 05him**
added cart and  wishlist page in the ecom app 

live URL :-https://05him.github.io/The_Dark_Store/

live URL for cart page:-https://05him.github.io/The_Dark_Store/src/cart/cart.html

live URL for wishlist page:- https://05him.github.io/The_Dark_Store/src/wishlist/wishlist.html

### CODE REVIEW

**Review by Developers-codz — APPROVED** [review]
Looks good to merge

**Review by saurabhchirde — APPROVED** [review]
Clean code with the proper naming convention, looks good to merge.

File: src/cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="cart-bill">
+                <div class="card-box-vertical card-text-only">
+                    <div class="card-text">
+                    <h3 class="card-title">Tension Free Jacket</h3>
+                    <hr>
+                    <div class="flex-center">Price(2items) <span class="cart-bill-price">₹4999</span></div>
```
Comment (SalonSikdar) [inline src/cart/cart.html:80]: 2<space>Items will make the UX better.

File: src/cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                    <div class="flex-center">Discount <span class="cart-bill-price">-₹4999</span></div>
+                    <div class="flex-center">Deleivery Charges<span class="cart-bill-price">₹10</span></div>
```
Comment (SalonSikdar) [inline src/cart/cart.html:82]: Typo - Delivery. Try a spell checker extension in your IDE for identifying typos.

File: src/wishlist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                        <h3 class="card-title">Tension Free Jacket</h3>
+                        <span class="card-price-final">₹2500</span>
+                        <span class="card-price-initial">₹5000</span>
+                        <span class="card-discount block">50% OFF</span>
+                        <span class="card-desc block">Hey This Is A Tensiosn Free Jacket While Wearing This You Will Go All Tension Free, So Buy Now!</span>
+                        <div class="card-btn-box flex-center">
+                            <button type="button" class="btn btn-primary block">Add To Cart</button>
+                            <button type="button" class="btn btn-secondary-outline block">Buy Now</button>
+                        </div>
+                    </div>
```
Comment (SalonSikdar) [inline src/wishlist/wishlist.html:107]: Try formatting with prettier for better indentation. Alt+shift+F on windows, Ctrl/Cmd+Shift+I for Mac. 
Logic looks clean and readable.

File: src/cart/cart.html (hunk 1 above)
Comment (05him) [reply src/cart/cart.html:80]: thanks for the suggestion , will correct in future updates, as I've already merged it without seeing  you review sorry for that

File: src/cart/cart.html (hunk 2 above)
Comment (05him) [reply src/cart/cart.html:82]: thank you! will use it from now

File: src/wishlist/wishlist.html (hunk 3 above)
Comment (05him) [reply src/wishlist/wishlist.html:107]: thanks for the short cut will try it out,  currently, my prettier was having some issues don't know why

**Pull request merged**